doc/user/content/third-party/debezium.md

```diff
@@ -15,7 +15,7 @@ Debezium emits records using an envelope that contains valuable information abou
 Currently, Materialize only supports Avro-encoded Debezium records. If you're interested in JSON support, please reach out in the community Slack or leave a comment on [this GitHub issue](https://github.com/MaterializeInc/materialize/issues/5231).
 {{</ note >}}
 
-### Database settings
+## Database settings
```

Comment:
Maybe it makes sense to just direct people to the CDC guides here. I'm going to push the MySQL one in the next few days, so I can also revamp this page as part of that PR.